definitions/npm/deepmerge_v4.x.x/flow_v0.100.x-/test_deepmerge_v4.x.x.js

```diff
@@ -0,0 +1,8 @@
+const deepmerge = require("deepmerge");
+
+const m1: { a: string, b: string, ... } = deepmerge({a: ''}, { b: '' })
```

Comment:
Tests needs to be [wrapped with `describe / it` blocks](https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#use-describe-and-it-blocks-to-limit-scope).